Tests/fixtures/podcasting.yml

```diff
@@ -0,0 +1,9536 @@
+    version: "561"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Zeno'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Zeno
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Zune/4.8'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Zune
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Libsyn4-peek'
+  os: [ ]
+  client: null
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'UCast/1 CFNetwork/1406.0.4 Darwin/22.4.0'
+  os:
+    name: iOS
+    version: "16.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: UCast
+    version: "1"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'NPROneAndroid'
+  os: [ ]
+  client:
+    type: mobile app
+    name: NPR One
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'NPR%20One/371 CFNetwork/1399 Darwin/22.1.0'
+  os:
+    name: iOS
+    version: "16.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: NPR One
+    version: "371"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Trackable/0.1 NNjCeA'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Trackable
+    version: "0.1"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Uforia/1056 CFNetwork/1406.0.4 Darwin/22.4.0'
+  os:
+    name: iOS
+    version: "16.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: Uforia
+    version: "1056"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'LAT-Native-App; iPhone; iOS 15.7.3'
+  os:
+    name: iOS
+    version: "15.7.3"
+    platform: ""
+  client:
+    type: mobile app
+    name: L.A. Times
+    version: ""
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Mozilla/5.0 (iPhone; CPU iPhone OS 16_1_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) nytios/9.91.1'
+  os:
+    name: iOS
+    version: "16.1.2"
+    platform: ""
+  client:
+    type: mobile app
+    name: NYTimes
+    version: "9.91.1"
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'LiSTNR.au.com.podcastoneaustralia/3.24.1 iPhone12,1 iOS/16.5.1 CFNetwork/1408.0.4 Darwin/22.5.0'
+  os:
+    name: iOS
+    version: "16.5.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: LiSTNR App
```

Comment:
this refers https://apps.apple.com/au/app/listnr/id1462845202 and https://play.google.com/store/apps/details?id=au.com.podcastoneaustralia&hl=en_AU and looks like official name does not have "App" in it, so it should be just "LiSTNR"